tests/performanceplatform/utils/test_collector_config.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+from hamcrest import assert_that, equal_to
+from mock import patch
+from performanceplatform.utils.collector import get_config
+
+
+class ConfigTest(unittest.TestCase):
+    @patch('performanceplatform.client.collector.CollectorAPI.get_collector')
```

Comment:
What are we actually testing in this test? The [method doesnt seem to be doing much](https://github.com/alphagov/performanceplatform-collector/pull/126/files#diff-9da4ce370d7a41fc2d891a4afa01eea6R4) so I'm not sure if we need a test